### PR TITLE
Add the variables description and groups in application connector workflows

### PR DESCRIPTION
**Pull request by aminelouati**

### CODE REVIEW

File: ApplicationConnectors/resources/catalog/IBM_InfoSphere_DataStage.xml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <job xmlns="urn:proactive:jobdescriptor:3.13" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" maxNumberOfExecution="2" name="IBM_InfoSphere_DataStage" onTaskError="continueJobExecution" priority="normal" projectName="ETL" xsi:schemaLocation="urn:proactive:jobdescriptor:3.13 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.13/schedulerjob.xsd">
   <variables>
-    <variable name="IS_ENDPOINT" value="https://SERVICE-TIER-HOSTNAME:IS-Port/ibm/iis/api/dscdesignerapi"/>
-    <variable name="IS_USER" value=""/>
-    <variable model="PA:HIDDEN" name="IS_PASSWORD" value=""/>
-    <variable name="IS_API" value=""/>
-    <variable name="IS_API_VERSION" value=""/>
+    <variable name="IS_ENDPOINT" value="https://SERVICE-TIER-HOSTNAME:IS-Port/ibm/iis/api/dscdesignerapi" description="The server base url"/>
+    <variable name="IS_USER" value=""  description="Basic auth user"/>
```
Comment (mklkun) [inline ApplicationConnectors/resources/catalog/IBM_InfoSphere_DataStage.xml:5]: auth => authentification ?

File: ApplicationConnectors/resources/catalog/IBM_InfoSphere_DataStage.xml [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+    <variable model="PA:HIDDEN" name="IS_PASSWORD" value="" description="Basic auth password"/>
```
Comment (mklkun) [inline ApplicationConnectors/resources/catalog/IBM_InfoSphere_DataStage.xml:6]: same

File: ApplicationConnectors/resources/catalog/Centreon_Notification.xml [hunk 3]
```diff
@@ -1,25 +1,18 @@
...
   <variables>
-    <variable name="CENTREON_ENDPOINT" value="https://server_hostname/centreon/api"/>
-    <variable name="CENTREON_USER" value=""/>
-    <variable model="PA:HIDDEN" name="CENTREON_PASSWORD" value=""/>
-    <variable name="CENTREON_NOTIF_TIMESTAMP" value=""/>
-    <variable name="CENTREON_NOTIF_HOST" value=""/>
-    <variable name="CENTREON_NOTIF_SERVICE" value=""/>
-    <variable model="PA:LIST(0,1,2)" name="CENTREON_NOTIF_STATUS" value="0"/>
-    <variable name="CENTREON_NOTIF_MESSAGE" value=""/>
+    <variable name="CENTREON_ENDPOINT" value="https://server_hostname/centreon/api" description="The server base url" group="Centreon Connection"/>
```
Comment (mklkun) [inline ApplicationConnectors/resources/catalog/Centreon_Notification.xml:4]: You did not add hidden and advanced information. Is it on purpose?

File: ApplicationConnectors/resources/catalog/JD_Edwards_Orchestrator.xml [hunk 4]
```diff
@@ -1,30 +1,22 @@
...
-    <variable name="JDE_USER" value="demo"/>
-    <variable model="PA:HIDDEN" name="JDE_PASSWORD" value="ENC(B8nLpbVTAvupPzLkccUo/g==)"/>
-    <variable name="JDE_ENVIRONMENT" value="JDV920"/>
-    <variable name="JDE_ROLE" value="*ALL"/>
-    <variable name="JDE_DEVICE" value="Curl"/>
-    <variable name="JDE_ORCHESTRATION" value=""/>
+    <variable name="JDE_ENDPOINT" value="https://ais_server:port" description="AIS Server base URL" group="JD Edwards Connection"/>
+    <variable name="JDE_USER" value="" description="Basic auth user" group="JD Edwards Connection"/>
+    <variable model="PA:HIDDEN" name="JDE_PASSWORD" value="" description="Basic auth password" group="JD Edwards Connection"/>
+    <variable name="JDE_ENVIRONMENT" value="JDV920" description="Specify the environment to use for login. If not specified, the default environment configured for the AIS Server will be used" group="JD Edwards Parameters"/>
```
Comment (mklkun) [inline ApplicationConnectors/resources/catalog/JD_Edwards_Orchestrator.xml:7]: for login => to login (maybe?)

File: ApplicationConnectors/resources/catalog/JD_Edwards_Orchestrator.xml [hunk 5]
```diff
@@ -1,30 +1,22 @@
...
+    <variable name="JDE_ROLE" value="*ALL" description="The role to use for login. If not specified, the default role configured for the AIS Server will be used" group="JD Edwards Parameters"/>
```
Comment (mklkun) [inline ApplicationConnectors/resources/catalog/JD_Edwards_Orchestrator.xml:8]: same

File: ApplicationConnectors/resources/catalog/JD_Edwards_Server_Manager.xml [hunk 6]
```diff
@@ -1,20 +1,13 @@
...
 <job xmlns="urn:proactive:jobdescriptor:3.13" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" maxNumberOfExecution="2" name="JD_Edwards_Server_Manager" onTaskError="continueJobExecution" priority="normal" projectName="ERP" xsi:schemaLocation="urn:proactive:jobdescriptor:3.13 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.13/schedulerjob.xsd">
   <variables>
-    <variable name="JDE_ENDPOINT" value="https://ais_server:port/manage/mgmtrestservice"/>
-    <variable name="JDE_USER" value="demo"/>
-    <variable model="PA:HIDDEN" name="JDE_PASSWORD" value="ENC(B8nLpbVTAvupPzLkccUo/g==)"/>
-    <variable name="JDE_ENVIRONMENT" value="JDV920"/>
-    <variable name="JDE_ROLE" value="*ALL"/>
-    <variable name="JDE_DEVICE" value="Curl"/>
+    <variable name="JDE_ENDPOINT" value="https://ais_server:port/manage/mgmtrestservice" description="AIS Server base URL" group="JD Edwards Connection"/>
+    <variable name="JDE_USER" value="" description="Basic auth user" group="JD Edwards Connection"/>
```
Comment (mklkun) [inline ApplicationConnectors/resources/catalog/JD_Edwards_Server_Manager.xml:5]: auth => authentification?

File: ApplicationConnectors/resources/catalog/JD_Edwards_Server_Manager.xml [hunk 7]
```diff
@@ -1,20 +1,13 @@
...
+    <variable model="PA:HIDDEN" name="JDE_PASSWORD" value="" description="Basic auth password" group="JD Edwards Connection"/>
```
Comment (mklkun) [inline ApplicationConnectors/resources/catalog/JD_Edwards_Server_Manager.xml:6]: same

File: ApplicationConnectors/resources/catalog/PeopleSoft.xml [hunk 8]
```diff
@@ -3,10 +3,10 @@
...
   <variables>
-    <variable name="ENDPOINT" value="https://peoplesoft_server_url/PSIGW/RESTListeningConnector/PSFT_XX/C_REST_ECHO.v1/echo" model="PA:URL"   advanced="false" hidden="false" description="base url of the request (Inherited from the WF variables)"/>
-    <variable name="USERNAME" value="" description="Basic auth user default value is empty (Inherited from the WF variables)"/>
-    <variable name="PASSWORD" value="" model="PA:Hidden" description="Basic auth password default value is empty (Inherited from the WF variables)"/>
-    <variable name="JWT_TOKEN" value="" model="PA:Hidden" description="Alternative authentication method (Inherited from the WF variables)"/>
+    <variable name="ENDPOINT" value="https://peoplesoft_server_url/PSIGW/RESTListeningConnector/PSFT_XX/C_REST_ECHO.v1/echo" model="PA:URL"   advanced="false" hidden="false" description="Base url of the request"/>
+    <variable name="USERNAME" value="" description="Basic auth user"/>
```
Comment (mklkun) [inline ApplicationConnectors/resources/catalog/PeopleSoft.xml:7]: same

File: ApplicationConnectors/resources/catalog/PeopleSoft.xml [hunk 9]
```diff
@@ -3,10 +3,10 @@
...
+    <variable name="PASSWORD" value="" model="PA:Hidden" description="Basic auth password"/>
```
Comment (mklkun) [inline ApplicationConnectors/resources/catalog/PeopleSoft.xml:8]: same

File: ApplicationConnectors/resources/catalog/Centreon_Notification.xml (hunk 3 above)
Comment (aminelouati) [reply ApplicationConnectors/resources/catalog/Centreon_Notification.xml:4]: Yes on purpose as all these variables are mandatory

**Pull request merged**